src/interfaces/python/swig_typemaps.i

```diff
@@ -1417,7 +1417,7 @@ def _internal_get_param(self, name):
         except Exception:
             raise
     if name in self.parameter_names():
-        raise ValueError("The current Python API does not have a getter for '{}'".format(name))
+        raise ValueError("The current Python API does not have a getter for '{}' which has type '{}'".format(name, self.parameter_types()[name]))
     else:
```

Comment:
“Of type” maybe?

Reply:
yup, I had that at first but then for some reason sounded weird, but I agree that is better!